functions/fn_VAboxArsenalinit.sqf

```diff
@@ -0,0 +1,39 @@
+/*
+ * Author: JoramD
+ * Initializes 2 ace arsenals on an object (1 full, 1 partial)
+ *
+ * Arguments:
+ * 0: Crate <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [crate] call TAC_Olympus_fnc_VAboxArsenalinit
```

Comment:
`VAboxArsenalInit` (capitalized `Init`), file name as well of course and wherever it's called from.